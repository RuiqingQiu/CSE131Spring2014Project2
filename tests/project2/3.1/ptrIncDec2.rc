typedef int [2] DOUBLE;

function : int main(){
   DOUBLE d;
   auto p = &d;
   auto b = p;
   cout << (int)p++ - (int)b << endl;
   cout << sizeof(DOUBLE) << endl;
   cout << (int)p - (int)b << endl;
   cout << (int)++p - (int)b << endl;
   cout << (int)--p - (int)b << endl;
   cout << (int)p-- - (int)b << endl;
   cout << (int)p - (int)b << endl;
   return 0;
}
