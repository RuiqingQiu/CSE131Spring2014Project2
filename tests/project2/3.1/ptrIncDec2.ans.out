0
8
8
16
8
8
0
